implementations/npm/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "todotasks",
+  "version": "1.0.0",
+  "description": "Base project for new taskrunner configurations.",
+  "main": "index.js",
+  "scripts": {
+    "postinstall": "bower install",
+    "test": "node test/test.js",
+    "devbuild": "",
+    "dev": "",
+    "clean": "rm -rf dist/*",
```

Comment:
Should we strive for windows compatibility? `rm` should be changed to `rimraf` module if so